frontend/ui/widget/sortwidget.lua

```diff
@@ -301,30 +263,39 @@ function SortWidget:init()
         vertical_footer,
     }
     -- setup title bar
-    self.title_bar = SortTitleWidget:new{
+    self.title_bar = TitleBar:new{
+        width = self.dimen.w,
+        align = "left",
+        with_bottom_line = true,
+        bottom_line_color = Blitbuffer.COLOR_DARK_GRAY,
+        bottom_line_h_padding = padding,
         title = self.title,
-        width = self.item_width,
-        height = self.item_height,
-        sort_page = self,
+        title_face = self.title_face,
```

Comment:
Any caller using it ? Or for future usage ? In which case document it in `local SortWidget = InputContainer:new()`